### PR TITLE
test(config): reduce playwright pixel threshold to 0.005

### PR DESCRIPTION
**Pull request by liamdebeasi**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
  - Some docs updates need to be made in the `ionic-docs` repo, in a separate PR. See the [contributing guide](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#modifying-documentation) for details.
- [ ] Build (`npm run build`) was run locally and any changes were pushed
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

The Playwright test threshold is at 0.01 which means some small diffs are not being caught.

## What is the new behavior?

- Reduces the Playwright test threshold to 0.005.
- Fixed a flaky test for `input` where the focused indicator would sometimes be animating in when the screenshot was captured.
- Fixed a flaky test for `item` where the reorder indicator would not always be visible.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by sean-perkins — APPROVED** [review]
This looks good. We will probably need a new PR for feature-6.4 and feature-7.0 after this lands in those branches, so that the diff screenshots are using the latest pixel ratio.

**Comment by liamdebeasi** [comment]
I'll hold off on merging https://github.com/ionic-team/ionic-framework/pull/26382 until this PR is merged.

**Pull request merged**
